### DISCUSSION TITLE
Make different teams access the same servers

### DISCUSSION THREAD

**Discussion by Smartlinuxcoder** [coollabsio/coolify#3235] [Improvement Requests]
Right now, when someone logs in in a different team than mine, cannot access the projects, which is good, but they also can't access any resources/servers, and there is no way to share them between teams.

It would be cool to have an option for sharing servers between users/teams.

**Comment by maietta** [comment]
I second this.

**Comment by AlexCernov** [comment]
this would be great

**Comment by chintansatva** [comment]
this is very good feature, I have multiple teams and tech leads, and I would like to distribute the server between three teams. They should not be seeing each other's resources. Also, the API token should not allow access to any other team's resources.
